packages/react/src/themes/teams-dark/siteVariables.ts

```diff
@@ -19,7 +19,7 @@ export const brand02 = '#e2e2f6'
 export const brand04 = '#bdbde6'
 export const brand06 = '#a6a7dc'
 export const brand08 = '#8b8cc7'
-export const brand12 = brand
+export const brand12 = colors.primary[500]
```

Comment:
is there a plan to remove the `brand*` names from other themes?
that would help A LOT with the `color` prop story since we won't need any code changes in the styles of the components for other themes (you probably remember the `Menu` mayhem)

Reply:
Yep, I remember the problems we had there. In my opinion, ideally we will have colors.primary[500] as a color used for the same thing in all themes, but the themes would change what primary[500] means in that context (it might be lighter in dark theme for example...) Not certain 100% for this yet, we have to have the whole picture first..

Reply:
This one is actually great example that, in dark theme, `colors.primary[500]` is used for `brand12`, although in the light theme it is used as `brand`. We should rearrange the colors primary in dark theme to match these values accordingly by specifying the correct hex values (I guess the values will be lighter then they are in the light theme)